packages/frontend/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+This document provides detailed instructions on contributing to the Metrom
+frontend, which is built with Next.js and uses `pnpm` as the package manager.
+
+## Getting started
+
+To begin, install the package dependencies using `pnpm`. Run the following
+command from the root of the project:
+
+```
+pnpm install
+```
+
+### Building the packages
+
+The frontend depends on internal packages (`sdk`, `ui`), so ensure they are
+built by running:
+
+```
+pnpm build
+```
+
+### Setting up unvironment uariables
```

Comment:
Typo here, should be environment

Reply:
Fixed in c0dc1e341febba4af6cce425f572e802f8a851c8.